src/main/java/picard/analysis/replicates/CollectIndependentReplicateMetrics.java

```diff
@@ -159,6 +160,13 @@ protected int doWork() {
         IOUtil.assertFileIsReadable(VCF);
         IOUtil.assertFileIsReadable(INPUT);
 
+        // get an iterator to reads that overlap the heterozygous sites
+        final SamReader in = SamReaderFactory.makeDefault().open(INPUT);
+
+        if (!in.hasIndex()) {
```

Comment:
Shouldn't this (and the IOUtil.assertFile...) be in a CustomCommandLineValidation method?

Reply:
the problem is that figuring out if a file has an index involved opening a reader against it...which can be time consuming (think about doing it over NIO)...so I figured I prefer only to open the file once when creating the reader...